packages/console/src/components/AppContent/components/UserInfo/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { useLogto } from '@logto/react';
+import { User, UserRole } from '@logto/schemas';
+import classNames from 'classnames';
+import React, { useEffect, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import Dropdown, { DropdownItem } from '@/components/Dropdown';
+import { getAvatarById } from '@/consts/avatars';
+import useApi from '@/hooks/use-api';
+import SignOut from '@/icons/SignOut';
+
+import * as styles from './index.module.scss';
+
+const UserInfo = () => {
+  const { isAuthenticated, getIdTokenClaims } = useLogto();
+  const api = useApi();
+  const navigate = useNavigate();
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const anchorRef = useRef<HTMLDivElement>(null);
+  const [showDropDown, setShowDropdown] = useState(false);
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    (async () => {
+      if (isAuthenticated) {
+        const { sub: userId } = getIdTokenClaims() ?? {};
+
+        if (userId) {
+          const data = await api.get(`/api/users/${userId}`).json<User>();
```

Comment:
This API may be protected with a role type.
Is it suitable if we request the user info from this API directly?

Reply:
agreed. Getting current user info should not be guarded by the admin role.  could use oidc/me API. `/api/users/${userId}` is for admin user use only.

Reply:
Good call, guys. I'll update this part

Reply:
Guys, due to the limitation of our `oidc/me` API response at this moment, I will keep the `/users/:userId` usage for now.
I have created a follow up PR to ditch the `users` API usage here, and replace it with `fetchUserInfo` SDK method once we extend the response data from `/oidc/me`.

https://linear.app/silverhand/issue/LOG-2815/replace-useruserid-api-with-fetchuserinfo